Tests/AssemblerLib/SteadyDiffusion2DExample1.h

```diff
@@ -140,7 +140,7 @@ struct SteadyDiffusion2DExample1
 	static const std::size_t mesh_stride = mesh_subdivs + 1;
 	static const std::size_t dim_eqs = mesh_stride * mesh_stride;
 	MeshLib::Mesh* msh;
-	std::vector<std::size_t> vec_DirichletBC_id;
+	std::vector<int> vec_DirichletBC_id;
```

Comment:
Should it be some `IndexType`? But in my opinion, it's a test. As long as it works, it's fine. If will fail due to the index type, we can easily fix it later on.